### DISCUSSION TITLE
#help

### DISCUSSION THREAD

**Discussion by raniagomaa123456** [community/community#68286] [GitHub Education]
### Hi everyone,

I have already been approved for the jet Hub student package, so whenever I try to book a Mongo certificate, this message appears to me

Welcome and thanks for logging in. Unfortunately we are unable to verify your status as a participant in the GitHub Student Developer Pack. Please visit the GitHub Student Pack home for additional information and how to register.

Approved Submitted Aug 31, 2023
Congrats! Your request for [@raniagomaa123456](https://github.com/raniagomaa123456) was approved and your benefits are now available.

**Comment by coincatcher22** [comment]
If you have already been approved for the GitHub Student Developer Pack and are encountering an error message when trying to book a Mongo certificate, there are a few steps you can take to resolve the issue:

Double-check your eligibility: Ensure that you meet all the eligibility criteria for the GitHub Student Developer Pack. Make sure you are a student and have an active GitHub account.

Verify your approval status: Confirm that your approval for the GitHub Student Developer Pack is still valid. Check the date of your approval and ensure that it has not expired.

Contact support: If you believe there is an error or discrepancy with your approval status, reach out to the support team of the platform or service you are trying to book the Mongo certificate from. Provide them with the details of your approval and explain the issue you are facing. They should be able to assist you further and help resolve the problem.

Visit the GitHub Student Pack home: As mentioned in the error message, visit the GitHub Student Pack home for additional information and instructions on how to register. It's possible that there may be additional steps or requirements that need to be fulfilled before you can access certain benefits, such as the Mongo certificate.

I hope you are doing well.

**Reply by samus-aran** [reply]
Hi @midas-1119 ,

Thanks so much for participating in the Community! While we encourage non-AI answers to questions in community, we also love experimenting and using new technologies. We ask if you do use ChatGPT or similar programs, please disclose what software and model version number you used to generate the response you used in your reply in order to promote transparency in the community.

ChatGPT answers in a tone that sounds like a tech support professional and while we support trying out this software, it is important to verify **that they are solutions before posting** otherwise it may mislead users into thinking that they are receiving official responses when they are not.

Per ChatGPT's website, the program does have limitations like:
- "May occasionally generate incorrect information"
- "May occasionally produce harmful instructions"
- "Limited knowledge of world and events after 2021"

Psst: There is a explicit Education Support option for users that are having trouble with redeeming the partner's benefits. Perhaps include that in your response if applicable.

**Comment by samus-aran** [comment]
Hi @raniagomaa123456  👋 

We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. But we're here to assist ⭐ 

I can share that your request has been passed onto the GitHub Education team and will be processed as soon as possible, but I am afraid I do not have a timeline to share for when you can expect this to happen.

I'll provide an update either on your ticket(s) or here when I have one 👍
